freqtrade/pairlist/ShuffleFilter.py

```diff
@@ -0,0 +1,50 @@
+"""
+Shuffle pair list filter
+"""
+import logging
+import random
+from typing import Dict, List
+
+from freqtrade.pairlist.IPairList import IPairList
+
+
+logger = logging.getLogger(__name__)
+
+
+class ShuffleFilter(IPairList):
```

Comment:
Although i lack ideas for a better name - isn't `ShuffleFilter` misleading? i mean - it doesn't filter ...?

Reply:
We already have *PairList and *Filter (which is also not absolutely necessary in my opinion), do you want to create the third entity, which won't add anything besides different suffix used in the name?

Filter is not necessary changes the number of the pairs in the pairlist -- it can also simply rearrange the pairs, sorting them in some order, by spread, by volume, etc. Or shuffling, like this one.

If you wish, the `number_assets` parameter can be added into this filter (and to all others as well, allowing each filter to limit the number of pairs in its resulting pairlist passed further in the chain)

Reply:
leave as is 👍 
it was just something i noticed...